host/4/bookworm/python/python312/python312.Dockerfile

```diff
@@ -0,0 +1,102 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+# Build the runtime from source
+ARG HOST_VERSION=4.34.2
```

Comment:
please use OOP host version 4.1035.1

cc @kshyju @fabiocav @vrdmr @CooperLink

Reply:
4.1036.0

Reply:
Updated to 4.1036.0